Doc/library/asyncio-task.rst

```diff
@@ -654,16 +654,16 @@ Timeouts
     If ``long_running_task`` takes more than 10 seconds to complete,
     the context manager will cancel the current task and handle
     the resulting :exc:`asyncio.CancelledError` internally, transforming it
-    into an :exc:`asyncio.TimeoutError` which can be caught and handled.
+    into an :exc:`TimeoutError` which can be caught and handled.
```

Comment:
Also change `an` into `a`, throughout the PR.

Reply:
Done ,Thanks